examples/minimal-winit/Cargo.toml

```diff
@@ -2,17 +2,16 @@
 name = "minimal-winit"
 version = "0.1.0"
 authors = ["Jay Oster <jay@kodewerx.org>"]
-edition = "2021"
+edition = "2024"
```

Comment:
The [MSRV policy](https://github.com/parasyte/pixels/blob/main/MSRV.md) requires use to support Rust 1.81. Edition 2024 is not available until 1.85.